core/src/message_queue.c

```diff
@@ -68,8 +77,10 @@ void MESSAGE_QUEUE_destroy(MESSAGE_QUEUE_HANDLE handle)
         MESSAGE_HANDLE message;
         while((message = message_pop(mq)) != NULL)
```

Comment:
No. :)  MESSAGE_QUEUE_pop is the external function, message_pop is the static function that's used internally.